composer.json

```diff
@@ -25,7 +25,7 @@
     "mautic/core-lib": "^5.0"
   },
   "require-dev": {
-    "friendsofphp/php-cs-fixer": "^3.5",
+    "friendsofphp/php-cs-fixer": "^3.14",
```

Comment:
Why was this downgrade needed?

Reply:
Should be upgrade. I made this to trigger the composer update